libs/ledger-live-common/src/families/elrond/speculos-deviceActions.ts

```diff
@@ -210,19 +210,9 @@ export const acceptEsdtTransferTransaction: DeviceAction<Transaction, any> =
       {
         title: "Fee",
         button: "Rr",
-        expectedValue: ({ account, transaction }) =>
-          formatCurrencyUnit(
-            account.unit,
-            transaction.fees || new BigNumber(50000),
-            {
-              showCode: true,
-              disableRounding: true,
-              joinFragmentsSeparator: " ",
-            }
-          ).replace(/\s+/g, " "),
```

Comment:
Is this required? It didn't seem to fail on previous bot runs, and from what you described the amount displayed on the device is consistent with the one displayed on Live before the transaction is sent, right? Only the confirmed operation may end up with different fees than the optimistic one?

Reply:
Yes, only the confirmed operation can have different fees.